### ISSUE TITLE
[bug]Unable to remove false portal

### ISSUE THREAD

**Issue by yulinlin404** [MeowMC/PortalGun#19]
version：1.16.5-1.2.5
1.place a portal in the overworld
2. go to the nether and place the other portal in the nether
3. Almost this meaning
4. I can't break the portal in the overworld
5. When I return to the nether，I can place another yellow portal.

![2021-06-13_02 30 50](https://user-images.githubusercontent.com/49760859/121786117-7e827300-cbf0-11eb-9d0a-e3eda9a88a77.png)
![2021-06-13_02 34 06](https://user-images.githubusercontent.com/49760859/121786119-80e4cd00-cbf0-11eb-9f43-2b6cc09bfb04.png)

**Comment by MeowMC** [comment]
yeah I'm gonna fix that by making it so that it clears portals if you change dimensions. Also thanks for the translations

**Comment by ThePhantomPig791** [comment]
> yeah I'm gonna fix that by making it so that it clears portals if you change dimensions. Also thanks for the translations

Oh my god this is gonna be a lifesaver, these unremovable portals are so annoying on my server
